hs_file_types/forms.py

```diff
@@ -346,8 +346,8 @@ class OriginalCoverageForm(forms.Form):
     southlimit = forms.DecimalField(label='South Extent', widget=forms.TextInput())
     westlimit = forms.DecimalField(label='West Extent', widget=forms.TextInput())
     units = forms.CharField(max_length=100, label='Extent Unit')
-    projection_string_type = forms.ChoiceField(choices=PRO_STR_TYPES,
-                                               label='Coordinate String Type', required=False)
+    projection_string_type = forms.CharField(max_length=100, label='Coordinate String Type', required=False,
+                                             widget=forms.TextInput())
```

Comment:
All the form fields are read only so max_length is not enforced as we are not allowing user input. However, since the corresponding model field has a max_length of 20 I am going to set it to 20.